### ISSUE TITLE
Check digital accessibility compliance

### ISSUE THREAD

**Issue by shigapov** [UB-Mannheim/UBi#111]
Find guidelines and check what should be done for digital accessibility compliance.

At Uni Mannheim there is a webpage: https://www.uni-mannheim.de/en/digital-accessibility/

We can also ask Neelam, I think, see: Neelam Vishen, Chen, T., & Kuper, S. (2025). Base4NFDI– Guidelines on Application of Web Accessibility (Version 1). Zenodo. https://doi.org/10.5281/zenodo.15719661

Tools: https://www.w3.org/WAI/test-evaluate/tools/list/
